build-system/pr-check/build-targets.js

```diff
@@ -178,11 +181,21 @@ const targetMatchers = {
         name.endsWith('.protoascii'))
     );
   },
+  'VALIDATOR_JAVA': (file) => {
+    return (
+      !isOwnersFile(file) &&
```

Comment:
ultra-nit: All the other matcher functions handle owners files in a separate `if` statement, in order to make subsequent updates less brittle (e.g. `VALIDATOR_WEBUI` below). Maybe do that here too?